### ISSUE TITLE
Sample weights can't be loaded in Python 3.4 with CatBoost 0.1.1.2

### ISSUE THREAD

**Issue by bonext** [catboost/catboost#192]
Part of error:
```
/home/bonext/miniconda3/envs/cp34/lib/python3.4/site-packages/catboost/core.py in _check_weight_shape(self, weight, data_len)
    221         if len(weight) != data_len:
    222             raise CatboostError("Length of weight={} and length of data={} are different.".format(len(weight), data_len))
--> 223         if not isinstance(weight[0], (int, float, long)):
    224             raise CatboostError("Invalid weight value type={}: must be 1 dimensional data with int, float or long types.".format(type(weight[0])))
    225 

NameError: name 'long' is not defined
```

Steps to reproduce:
https://gist.github.com/bonext/e83a051d9cf9981e70a713c01fcf656b

Info on the matter:
https://www.python.org/dev/peps/pep-0237/

TL;DR - in python3 there's no `long`

**Comment by annaveronika** [comment]
could you try updating? I think it should be fixed in newer versions.

**Comment by bonext** [comment]
Building from source is a bit tricky from inside a conda environment, but I will report how it goes.

**Comment by annaveronika** [comment]
this should be fixed by now

**Comment by bonext** [comment]
Yep. It works in catboost 0.6
Thanks!
